### PR TITLE
Update dependency versions of the whole project

### PR DESCRIPTION
**Pull request by GrigoriiBerezin**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.6")
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.6.1")
-addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.12")
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.0")
+addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.16")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0")
```
Comment (myazinn) [inline project/plugins.sbt]: Why did you downgrade it? Did you face issues with 1.6.1?
I don't mind downgrading version, I'm just curious.

Comment (GrigoriiBerezin) [reply project/plugins.sbt]: Probably missclick

Comment (GrigoriiBerezin) [reply project/plugins.sbt]: Done

**Review by myazinn — APPROVED** [review]

File: services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala [hunk 2]
```diff
@@ -17,7 +17,7 @@ import cromwell.pipeline.service.AuthorizationException.{
 import cromwell.pipeline.utils.{ AuthConfig, ExpirationTimeInSeconds }
 import org.scalamock.scalatest.MockFactory
 import org.scalatest.concurrent.ScalaFutures.whenReady
-import org.scalatest.{ Matchers, WordSpec }
+import org.scalatest.{ WordSpec, Matchers }
```
Comment (alesavin) [inline services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala:20]: nit: these excessive changes a little bit annoying =/

**Review by alesavin — APPROVED** [review]

File: services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala (hunk 2 above)
Comment (GrigoriiBerezin) [reply services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala:20]: Yes, I know, but some files are not formatted properly by scalafmt

**Pull request merged**